script.js

```diff
@@ -1,4 +1,183 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+// Version 2 Comfortable - (Mon - Thu: 2h - 3h)
+/*
+- Player input number of dice, number of player and then select game mode
+- Press start button to display the player and the player's optimal combination
+- The highest / Lowest combination wins
+*/
+const diceInput = document.querySelector("#dice-number");
+const playerInput = document.querySelector("#player-number");
+const gameModeInput = document.querySelector("#game-mode");
+const submitBtn = document.querySelector("#submit-button");
+const outputDiv = document.querySelector("#output-div");
+const resetBtn = document.querySelector("#reset-button");
+resetBtn.style.display = "none";
+
+let gameRecordArr = [];
+let playerCounter = 1;
+let diceRollMsg, winner, winnerMsg;
+let displayMsg = "";
+let numsArr = "";
+let gameBoard;
+
+// Toggle Buttons
+function toggleBtn() {
+  if (resetBtn.style.display == "none") {
+    submitBtn.style.display = "none";
+    resetBtn.style.display = "block";
+  } else {
+    submitBtn.style.display = "block";
+    resetBtn.style.display = "none";
+  }
+}
+
+// Submit button click event
+submitBtn.addEventListener("click", function () {
+  let result = main();
+  submitBtn.disable = true;
+  outputDiv.innerHTML = result;
+});
+
+// Reset button click event
+resetBtn.addEventListener("click", function () {
+  location.reload();
+});
+
+// Roll dice
+function rollDice() {
+  let diceNum = Math.floor(Math.random() * 6) + 1;
+  return diceNum;
+}
+
+// Bubble Sort: Array low --> high or high --> low
+function sortDiceNumArr(arr, mode) {
+  const swap = (arr, index1, index2) => {
+    [arr[index1], arr[index2]] = [arr[index2], arr[index1]];
+  };
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < arr.length - i; j++) {
+      if (arr[j] < arr[j + 1] && mode == "big") {
+        swap(arr, j, j + 1);
+      } else if (arr[j] > arr[j + 1] && mode == "small") {
+        swap(arr, j, j + 1);
+      }
+    }
+  }
+  return arr;
+}
```

Comment:
nice use of bubble sort as you mentioned when i went into your breakout room, and its great that you're implementing your own custom sort method but for simple use cases like this, try leveraging the power of the built-in sort function

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

for number arrays you need to provide a comparison function, the documentation touches a bit on that, do look for me if you have any qns on how to use it